internal/policy/root.go

```diff
@@ -27,6 +27,49 @@ func InitializeRootMetadata(key *tuf.Key) *tuf.RootMetadata {
 	return rootMetadata
 }
 
+func AddRootKey(rootMetadata *tuf.RootMetadata, rootKey *tuf.Key) *tuf.RootMetadata {
+	rootMetadata.Keys[rootKey.KeyID] = rootKey
+	if _, ok := rootMetadata.Roles[RootRoleName]; !ok {
+		return InitializeRootMetadata(rootKey)
+	}
+
+	rootMetadata.AddKey(rootKey)
+
+	rootRoles := rootMetadata.Roles[RootRoleName]
+
+	for _, keyID := range rootRoles.KeyIDs {
+		if keyID == rootKey.KeyID {
+			return rootMetadata
+		}
+	}
+
+	rootRoles.KeyIDs = append(rootRoles.KeyIDs, rootKey.KeyID)
+
+	rootMetadata.Roles[RootRoleName] = rootRoles
+
+	return rootMetadata
+}
+
+func RemoveRootKey(rootMetadata *tuf.RootMetadata, keyID string) (*tuf.RootMetadata, error) {
```

Comment:
For consistency with targets key, I wonder if this should be `Delete`.

Reply:
Changed